src/Surface.cpp

```diff
@@ -23,12 +23,27 @@ int surf_id() {
 /**
  * @brief Resets the auto-generated unique Surface ID counter to 10000.
  */
-void reset_surf_id() {
+void reset_surface_id() {
   auto_id = 10000;
```

Comment:
Could we either:
a) Have an argument detailing the reset value of ``auto_id``
OR
b) Make 10000 a constant such as ``DEFAULT_INIT_ID`` or some other name